### DISCUSSION TITLE
github.dev not working

### DISCUSSION THREAD

**Discussion by RexRowan** [community/community#43038] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I pressed '.' and can not access the web editor
![issue](https://user-images.githubusercontent.com/4164511/210168024-5b216082-1533-4409-92bb-1c7fb1596e7f.jpg)

**Comment by eabramsk** [comment, marked as answer]
@RexRowan I had the same issue, it works for me today.
Try to check again.

**Comment by jfullstackdev** [comment]
yup, it's already working by now

**Comment by RexRowan** [comment]
Ok guys, thanks for noticing me!
